model/types.go

```diff
@@ -65,34 +66,34 @@ const memWorking = "memory-working"
 const fsLimit = "fs-limit"
 const fsUsage = "fs-usage"
 
-// Request Types.
-// Used as parameters to all the Get methods of the model.
-type ClusterRequest struct {
+// epsilon values for the underlying in-memory stores
+const cpuLimitEpsilon = 10
```

Comment:
The same units as the values for that specific metric.
It's millicores for all CPU metrics and bytes for all other metric. Should I document that?